algebras/algebra/src/test/scala/endpoints4s/algebra/client/EncodeTestSuite.scala

```diff
@@ -0,0 +1,172 @@
+package endpoints4s.algebra.client
+
+import java.util.UUID
+
+import endpoints4s.algebra
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import endpoints4s.{Invalid, Valid}
+
+trait EncodeTestSuite[T <: algebra.Endpoints] extends AnyWordSpecLike with Matchers {
```

Comment:
This is not super important, but I think a name like `UrlEncodingTestSuite` or `UrlsTestSuite` would be clearer.